change/@ni-nimble-angular-204fcb18-d5bb-4ad9-bb52-53bda9837f75.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you also update the version listed in the `nimble-angular` README?

https://github.com/ni/nimble/blob/main/angular-workspace/projects/ni/nimble-angular/README.md#angular-support-policy